Source/WebCore/css/CSSProperties.json

```
@@ -3821,6 +3821,169 @@
                 "url": "https://www.w3.org/TR/css-content-3/#content-property"
             }
         },
+        "corner-shape": {
+            "animation-type": "discrete",
+            "initial": "round",
+            "codegen-properties": {
+                "longhands": [
+                    "corner-top-left-shape",
+                    "corner-top-right-shape",
+                    "corner-bottom-right-shape",
+                    "corner-bottom-left-shape"
+                ],
+                "disables-native-appearance": true,
+                "settings-flag": "cssCornerShapeEnabled"
+            },
+            "specification": {
+                "category": "css-borders",
+                "url": "https://drafts.csswg.org/css-borders-4/#corner-shaping"
+            }
+        },
+        "corner-top-left-shape": {
+            "animation-type": "discrete",
+            "initial": "round",
+            "codegen-properties": {
+                "disables-native-appearance": true,
+                "logical-property-group": {
+                    "name": "corner-shape",
+                    "resolver": "top-left"
+                },
+                "parser-grammar": "round | scoop | bevel | notch | straight",
```

```suggestion
                "parser-grammar": "<corner-shape-value>",
```

You can just refer to `<corner-shape-value>` directly, ditto below